release-process-documentation/documentation-guides/update-release-docs-new.md

```diff
@@ -0,0 +1,171 @@
+
+## Managing the cycle
+
+The designated "release meister" (aka Docs Lead), voluntold from [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers), is responsible for managing the process on the docs side and releasing the documentation on [kubernetes.io](https://kubernetes.io/) when the Kubernetes binary release goes out.
+
+### Relevant links
+
+*   k8s.io website - [https://kubernetes.io/](https://kubernetes.io/)
+*   k8s.io (docs) GitHub repo - [kubernetes/kubernetes.github.io](https://github.com/kubernetes/kubernetes.github.io)
+*   k8s.io [pull request (PR) queue](https://github.com/kubernetes/kubernetes.github.io/pulls) 
+*   OSS Kubernetes features tracking spreadsheet
+    *   [Kubernetes Features OSS tracking board (1.7 release)](https://docs.google.com/spreadsheets/d/1IJSTd3MHorwUt8i492GQaKKuAFsZppauT4v1LJ91WHY/edit?usp=sharing)
+    *   [Kubernetes Features OSS tracking board (1.8 release)](https://docs.google.com/spreadsheets/d/1AFksRDgAt6BGA3OjRNIiO3IyKmA-GU7CXaxbihy48ns/edit#gid=0)
+    *   Not sure whether there will be a tracking spreadsheet for 1.9
+*   Netlify dashboard - [https://app.netlify.com/](https://app.netlify.com/)
+*   [kubernetes.slack.com #sig-release](https://kubernetes.slack.com/messages/C2C40FMNF/) Slack channel
+
+### Start the cycle
+
+*   Create a new GitHub branch from `master` using the naming convention `release-X.Y`, e.g. `release-1.9`. Here's the [release-1.9 branch](https://github.com/kubernetes/kubernetes.github.io/tree/release-1.9).
+*   Update the Netlify _vnext staging_ site to use the new branch:
+    *   Login to [Netlify](https://app.netlify.com/) and navigate to the Sites tab.
+    *   Go to the settings for the [kubernetes-io-vnext-staging](https://app.netlify.com/sites/kubernetes-io-vnext-staging/settings) site.
+    *   Edit the Deploy settings and change "Branch" to the new version branch, e.g `release-1.9`.
+    *   Save the change and verify that it is live at [https://kubernetes-io-vnext-staging.netlify.com/](https://kubernetes-io-vnext-staging.netlify.com/).
+*   Add a reminder in the [pull request template](https://github.com/kubernetes/kubernetes.github.io/blob/master/.github/PULL_REQUEST_TEMPLATE.md) for PRs going into the new release to use the release base branch and set the Milestone, e. g. [this commit](https://github.com/kubernetes/kubernetes.github.io/pull/4632).
+*   Connect with the release manager -- introduce yourself, etc.
+*   Join these groups: [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release), [kubernetes-sig-leads](https://groups.google.com/forum/#!forum/kubernetes-sig-leads), [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev).
```

Comment:
https://groups.google.com/forum/#!forum/kubernetes-pm for the features-related activities.